closed/OpenJ9.gmk

```diff
@@ -333,11 +336,13 @@ run-preprocessors-j9 : stage-j9 \
 build-j9 : run-preprocessors-j9
 	@$(ECHO) Compiling OpenJ9 in $(OUTPUT_ROOT)/vm
 ifeq ($(OPENJ9_ENABLE_CMAKE),true)
-	(export OMR_DIR=$(OUTPUT_ROOT)/vm/omr OPENJ9_BUILD=true && \
+	(export OPENJ9_BUILD=true $(EXPORT_MSVS_ENV_VARS) && \
```

Comment:
Did you mean to remove ```OMR_DIR=$(OUTPUT_ROOT)/vm/omr```

Reply:
Yes, I did. It is not required. @dnakamura please confirm, thank you!

Reply:
Correct. it is a vestigial variable